WooCommerce/Classes/ViewRelated/Upgrades/SubscriptionsView.swift

```diff
@@ -71,6 +64,26 @@ struct SubscriptionsView: View {
         .task {
             viewModel.loadPlan()
         }
+        .sheet(isPresented: $isShowingSupport) {
+            supportForm
+        }
+    }
+}
+
+private extension SubscriptionsView {
+    var supportForm: some View {
+        NavigationView {
+            SupportForm(isPresented: $isShowingSupport,
+                        viewModel: SupportFormViewModel())
+            .toolbar {
+                ToolbarItem(placement: .confirmationAction) {
+                    Button(Localization.done) {
+                        isShowingSupport = false
+                    }
+                }
+            }
+        }
+        .navigationViewStyle(.stack)
```

Comment:
Nit: Please replace the deprecated `NavigationView` with `NavigationStack`.

Reply:
Thanks, I updated SubscriptionsView and UpgradesView on https://github.com/woocommerce/woocommerce-ios/pull/13577/commits/89765130d79a35b6d102687a4106d41f4dcc3c1d

There are still other usages of the combination of `NavigationView` and `.navigationViewStyle(.stack)` in the app, but I didn't have time to smoke test all of them so I'm keeping them unchanged for now.